### DISCUSSION TITLE
How to change toggler position ? (dark mode)

### DISCUSSION THREAD

**Discussion by lukaszjenek** [luizdepra/hugo-coder#762] [General]
Hi

Dark mode toggler is location bottom, on the right side.
I want to display it, on the top for example.
How to do it ?

Lukas

**Comment by luizdepra** [comment]
Hey.

If you still want a floating button, you could change [these lines](https://github.com/luizdepra/hugo-coder/blob/main/assets/scss/_float.scss#L2-L3).
But, if you want to embed it into the top bar, it will be more complicated. hahaha
